LCLSGeneral/LCLSGeneral/DUTs/DUT_EPS.TcDUT

```diff
@@ -15,7 +15,7 @@ STRUCT
     {attribute 'pytmc' := '
         pv: sFlagDesc
         io: i
-        field: DESC of nFlag variable, using ';' seperated list.
+        field: DESC of nFlag variable, using ; separated list.
```

Comment:
`;` is treated as a field delimiter / newline equivalent

https://github.com/pcdshub/pytmc/blob/master/pytmc/pragmas.py#L20

Reply:
There's no escaping mechanism to my knowledge so yes that's your best option

Reply:
Something like
```suggestion
        field: DESC semicolon-delimited nFlag variable
```